app/src/main/java/com/kickstarter/ui/activities/compose/search/CategorySelectionSheet.kt

```diff
@@ -140,52 +136,16 @@ fun CategorySelectionSheet(
                     }
                 }
 
-                Row(
-                    modifier = Modifier
-                        .fillMaxWidth()
-                        .height(88.dp)
-                        .drawBehind {
-                            drawLine(
-                                color = backgroundDisabledColor,
-                                start = Offset(0f, 0f),
-                                end = Offset(size.width, 0f),
-                                strokeWidth = dimensions.dividerThickness.toPx()
-                            )
-                        },
-                    verticalAlignment = Alignment.CenterVertically
-                ) {
-                    Row(
-                        modifier = Modifier
-                            .fillMaxWidth()
-                            .padding(dimensions.paddingLarge),
-                        horizontalArrangement = Arrangement.spacedBy(dimensions.paddingMedium),
-                    ) {
-                        val resetCategoryName = stringResource(R.string.Category)
-                        KSOutlinedButton(
-                            modifier = Modifier
-                                .defaultMinSize(minHeight = dimensions.minButtonHeight),
-                            backgroundColor = colors.backgroundSurfacePrimary,
-                            textColor = colors.textPrimary,
-                            onClickAction = {
-                                selectedCategory.value = Category.builder().name(resetCategoryName).build()
-                            },
-                            text = stringResource(R.string.Reset_filters),
-                            isEnabled = !isLoading
-                        )
-                        KSButton(
-                            modifier = Modifier.weight(1f),
-                            backgroundColor = colors.kds_black,
-                            textColor = colors.kds_white,
-                            onClickAction = {
-                                selectedCategory.value?.let { onApply(it) }
-                            },
-                            shape = RoundedCornerShape(size = KSTheme.dimensions.radiusExtraSmall),
-                            text = stringResource(R.string.See_results),
-                            textStyle = typographyV2.buttonLabel,
-                            isEnabled = !isLoading,
-                        )
+                val resetCategoryName = stringResource(R.string.Category)
+                KSSearchBottomSheetFooter(
```

Comment:
This piece has been extended to be a re-usable component, and included in the Design system as one of the available footers